sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -108,7 +108,7 @@ export class SecretClient {
    *
    * let client = new SecretClient(vaultUrl, credentials);
    * ```
-   * @param vaultUrl - The base URL to the vault.
+   * @param vaultUrl - The base URL to the vault. You should validate that this URL references a valid Key Vault HSM resource. See https://aka.ms/azsdk/blog/vault-uri for details.
```

Comment:
No such thing as "Key Vault HSM". Be sure to delete "HSM". Should be "Key Vault or Managed HSM" for admin and keys, and only "Key Vault" for certs and secrets.

Reply:
Bah, this one was just me only deleting half of "Managed HSM" from the copy+paste by accident. These should all be fixed up now.